docs/openapi/open-api.yml

```diff
@@ -69,3 +138,68 @@ components:
         - mSettings
         - mStandings
         - mTeam
+        - players_wl
+        - proTeamSchedules_wl
+        - mDraftDetail
+    
+    TransactionPayload:
```

Comment:
Rename to something like "LineupChangePayload". Unless this can be used for other transactions as well.

Reply:
I had it lineup specific but I think you can have add/drop transactions as well

Reply:
I also wonder if this would work for trades too.